packages/venia-concept/src/components/Checkout/wrapper.js

```diff
@@ -92,33 +88,29 @@ class CheckoutWrapper extends Component {
 
         const actions = {
             beginCheckout,
+            cancelCheckout,
             editOrder,
             getShippingMethods,
             requestOrder,
-            resetCheckout,
             submitShippingAddress,
             submitOrder,
             submitPaymentMethodAndBillingAddress,
             submitShippingMethod
         };
 
         const { shippingMethods: availableShippingMethods } = cart;
-        const { paymentData, shippingMethod, shippingTitle } = checkout;
+        const { paymentData, shippingAddress, shippingMethod } = checkout;
 
         const miscProps = {
             availableShippingMethods,
+            havePaymentMethod: hasData(paymentData),
```

Comment:
I usually like `has` over `have`. Just... you know, incase you wanted to change it.

Reply:
I don't feel too strongly about it aside from the fact that the consuming code is doing `if (!havePaymentMethod)`, which - in english - reads nicer than `if (!hasPaymentMethod)` but I'm really splitting hairs here 😄

Reply:
I've never seen `haveData` before. @sirugh is right, IMO.

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/799/commits/bc395d699d381b578cd3d88bbc8b6ea35e728fca.